src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventAttachments/index.tsx

```diff
@@ -0,0 +1,156 @@
+};
+
+class GroupEventAttachments extends React.Component<Props, State> {
+  state = {eventAttachmentsList: [], loading: true, error: null, pageLinks: null};
+
+  componentWillMount() {
+    this.fetchData();
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    if (this.props.location.search !== prevProps.location.search) {
+      this.fetchData();
+    }
+  }
+
+  handleDelete = async (url: string) => {
+    const {api, params} = this.props;
+
+    this.setState({
+      loading: true,
+    });
+
+    try {
+      await deleteEventAttachment(api, url, params.groupId);
+      this.fetchData();
+    } catch (_err) {
+      // TODO: Error-handling
+    }
+  };
+
+  fetchData = () => {
+    this.setState({
+      loading: true,
+      error: null,
+    });
+
+    const query = {
+      ...pick(this.props.location.query, ['cursor', 'environment', 'types']),
+      limit: 50,
+    };
+
+    this.props.api.request(`/issues/${this.props.params.groupId}/attachments/`, {
+      query,
+      method: 'GET',
+      success: (data, _, jqXHR) => {
+        this.setState(prevState => ({
+          eventAttachmentsList: data,
+          error: null,
+          loading: false,
+          pageLinks: jqXHR ? jqXHR.getResponseHeader('Link') : prevState.pageLinks,
+        }));
+        GroupStore.updateEventAttachmentsCount(data.length, this.props.params.groupId);
+      },
+      error: err => {
+        this.setState({
+          error: parseApiError(err),
+          loading: false,
+        });
+      },
+    });
+  };
+
+  renderNoQueryResults() {
+    return (
+      <EmptyStateWarning>
+        <p>{t('Sorry, no event attachments match your search query.')}</p>
+      </EmptyStateWarning>
+    );
+  }
+
+  renderEmpty() {
+    return (
+      <EmptyStateWarning>
+        <p>{t("There don't seem to be any event attachments yet.")}</p>
+      </EmptyStateWarning>
+    );
+  }
+
+  renderResults() {
+    const {group, params} = this.props;
+
+    return (
+      <GroupEventAttachmentsTable
+        attachments={this.state.eventAttachmentsList}
+        orgId={params.orgId}
+        projectId={group.project.slug}
+        groupId={params.groupId}
+        onDelete={this.handleDelete}
+      />
+    );
+  }
+
+  renderBody() {
+    let body;
+
+    if (this.state.loading) {
+      body = <LoadingIndicator />;
+    } else if (this.state.error) {
+      body = <LoadingError message={this.state.error} onRetry={this.fetchData} />;
+    } else if (this.state.eventAttachmentsList.length > 0) {
+      body = this.renderResults();
+    } else if (this.props.location.query.types) {
+      body = this.renderNoQueryResults();
+    } else {
+      body = this.renderEmpty();
+    }
+
+    return body;
+  }
+
+  // TODO: do we need to feature flag protect this entire component/route, or is hiding only the navigation tab enough?
+
+  render() {
+    return (
+      <div>
+        <GroupEventAttachmentsFilter />
+        <Panel className="event-list">
```

Comment:
Are Panel and PanelBody needed here?

Reply:
Imho yes, because without it it doesn't look quite right:

![image](https://user-images.githubusercontent.com/9060071/69807296-3e61c800-11e5-11ea-9db7-4c99c44a2f4e.png)